torch/onnx/_internal/fx/export.py

```diff
@@ -415,6 +433,18 @@ def _export_fx_to_onnxscript(fx_module_with_metadata, opset_version):
             assert isinstance(output, (graph_building.TorchScriptTensor, tuple)), type(
                 output
             )
+            # op-level validation
+            # TODO(titaiwang): Change ORTEvaluator to ReferenceEvaluator
+            # Symbolic_fn should have the same output as node.target (torch ops)
+            with evaluator.default_as(evaluator.ORTEvaluator()):
+                expected_output = node.target(*torch_args, **torch_kwargs)
+                numpy_args = [
+                    arg.numpy() if isinstance(arg, torch.Tensor) else arg
+                    for arg in torch_args
+                ]
+                ort_output = symbolic_fn(*numpy_args, **torch_kwargs)
+                torch.testing.assert_close(expected_output.numpy(), ort_output)
```

Comment:
Non blocking. I wonder if we'd want to raise exception directly like this, or raise till all nodes are converted and tested.

A nice message is handy too, letting users know how to disable the check.

Reply:
I like the idea of accumulating the messages and store them. I guess that can wait until the diagnostic module. For now a warning seems appropriate so that we don't fail in the middle.